### PR TITLE
[PyTorch] NestedTensor kernels for {r,g}elu{,_}

### PR DESCRIPTION
**Pull request by swolchok**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #75493
* #75347
* #75346
* #75491
* __->__ #75370
* #74629
* #74490

These are simple element-wise ops it's convenient to be able to use with NestedTensor.

Differential Revision: [D35448205](https://our.internmc.facebook.com/intern/diff/D35448205/)

### CODE REVIEW

**Comment by swolchok** [comment]
FWIW I ended up not actually needing these for transformers, but I wrote them so here they are.

**Review by cpuhrsch — APPROVED** [review]
Thanks!

File: test/test_nestedtensor.py [hunk 1]
```diff
@@ -183,3 +183,13 @@ def test_repr_string(self):
         )
         self.assertEqual(str(a), expected)
         self.assertEqual(repr(a), expected)
+
+    @torch.inference_mode()
+    def test_activations(self):
+        for func in (torch.nn.functional.relu, torch.nn.functional.relu_, torch.nn.functional.gelu, torch._C._nn.gelu_):
+            t = torch.tensor([-1, 0, 1], dtype=torch.float)
+            nt = nested_tensor([t])
+            print(func)
```
Comment (swolchok) [inline test/test_nestedtensor.py:192]: this should not be here, will remove

**Comment by swolchok** [comment]
hey @cpuhrsch can you GH1 land this stack?

**Pull request closed without merging**